src/object/srv_internal.h

```diff
@@ -104,6 +104,43 @@ struct migrate_pool_tls {
 				mpt_fini:1;
 };
 
+struct obj_bulk_args {
+	ABT_eventual	eventual;
+	uint64_t	bulk_size;
+	int		bulks_inflight;
+	int		result;
+	bool		inited;
+};
+
+struct obj_tgt_query_args {
+	struct obj_io_context	*ioc;
+	struct dtx_handle	*dth;
+	daos_key_t		*in_dkey;
+	daos_key_t		*in_akey;
+	daos_key_t		*out_dkey;
+	daos_key_t		*out_akey;
+	daos_key_t		 dkey_copy;
+	daos_key_t		 akey_copy;
+	daos_recx_t		 recx;
+	daos_epoch_t		 max_epoch;
+	int			 result;
+	uint32_t		 shard;
```

Comment:
may add prefix like "tqa_" to the fields, not easy to grep when reading the code to understand what's shard number handling...

Reply:
OK, will do that in subsequent patch.